components/x-topic-search/src/lib/get-suggestions.js

```diff
@@ -3,39 +3,47 @@ const addQueryParamToUrl = (name, value, url, append = true) => {
 	return append === true ? `${url}&${queryParam}` : `${url}?${queryParam}`;
 };
 
-const suggest = function (suggestions, followedTopics, searchTerm) {
+const separateFollowedAndNot = (suggestions, followedTopics) => {
+	const matchingFollowedTopics = [];
+	const matchingUnfollowedTopics = [];
+
+	if (!followedTopic.length) {
+		return { matchingFollowedTopics, matchingUnfollowedTopics: suggestions }
+	}
+
+	followedTopics.forEach(followedTopic => suggestions.forEach(suggestion => {
+		if (suggestion.id === followedTopic.uuid) {
+			matchingFollowedTopics.push(suggestion);
+		} else {
+			matchingUnfollowedTopics.push(suggestion)
+		}
+	}));
+
+	return { matchingFollowedTopics, matchingUnfollowedTopics };
+}
+
+const suggest = function (suggestions, followedTopics) {
+
 	if (suggestions.length) {
 		suggestions.forEach((suggestion) => {
 			if (suggestion && !suggestion.url){
 				// TODO App needs different url?
 				suggestion.url = '/stream/' + suggestion.id;
```

Comment:
This only needs to be done to suggestions that will be rendered? If so, then perhaps this is something that should be done at the point of render in `SuggestionList.jsx`?

```jsx
				<a
					data-trackable="topic-link
					className={classNames(styles["suggestion__name"])}
					href={`/stream/${suggestion.url}`}
				>
					{ suggestion.prefLabel }
				</a>
```